package.json

```diff
@@ -70,36 +70,36 @@
     "htmlparser2": "^7.2.0",
     "localforage": "^1.10.0",
     "lodash": "4.17.21",
-    "metro-config": "^0.64.0",
+    "metro-config": "^0.71.3",
     "moment": "^2.29.4",
     "moment-timezone": "^0.5.31",
     "onfido-sdk-ui": "^8.1.0",
     "prop-types": "^15.7.2",
     "pusher-js": "^7.0.6",
-    "react": "^17.0.2",
+    "react": "18.0.0",
     "react-collapse": "^5.1.0",
     "react-content-loader": "^6.1.0",
     "react-dom": "^17.0.2",
```

Comment:
@janicduplessis @roryabraham 
Any reason for `react-dom` to stay on 17 while React is updated to 18?
Usually they have to be the same (major) version

Reply:
I wanted to avoid updating it as I'm not sure react-native-web is compatible yet. From our testing it works just fine. The react package doesn't really include much code, it is basically just a facade for react-dom / react-native renderers.

Reply:
Updating to react-dom 18 would be really nice to do in a follow up tho :)

Reply:
I'm going to continue working on updating our react-native-web fork, and we can update `react-dom` when we update NewDot to point at the updated `react-native-web` fork?